src/main/resources/ProcessingTools.json

```diff
@@ -0,0 +1,10 @@
+[
+  {
+    "processToolName": "furnace",
+    "processesNames": [
```

Comment:
Does this need to be entered twice (i.e. already given other way in Processes.json), why not just calculate it rather than increasing input data required.

Reply:
I think it is fine to remove the reference from processes.json

Reply:
Ok, will count as resolved and assume change is made.